.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "Analyzers"]
+	path = Analyzers
```

Comment:
Why not `Lombiq.Analyzers.PowerShell`?

Reply:
To fit in with the other directories in the repo. Should I change it? If yes, what to? Lombiq.Analyzers.PowerShell or Analyzers/Lombiq.Analyzers.PowerShell

Reply:
Let's do _tools/Lombiq.Analyzers.PowerShell_.

Reply:
but that stands out from the rest :(

Reply:
It also took me some time to get used to all the random lowercase-named folders in ASP.NET Core projects. It's still easier than having our own unique conventions :D.

Reply:
Ok, ugly it is!

Reply:
![image](https://user-images.githubusercontent.com/1976647/180309177-69e7266a-72ff-4c30-88ae-8c631f5e3dcd.png)

Reply:
aaand of course it totally broke and sent me off on a complimentary wild goose chase! smh

Reply:
Sending thoughts and prayers.